src/v/wasm/wasmtime.cc

```diff
@@ -599,7 +600,9 @@ class wasmtime_engine : public engine {
         // Poll the call future to completion, yielding to the scheduler when
         // the future yields.
         auto start = ss::steady_clock_type::now();
+        ss::internal::profiler_drop_stacktraces(true);
```

Comment:
Any reason you are not using the scoped RAII wrappers here?

Reply:
I guess I can use that for the outer-scope and then use the manual ones during the yields

Reply:
Yes it is - pushed this change. Thanks!

Reply:
yeh i spent the review time trying to convince myself those co_awaits wouldn't throw

Reply:
Yeah it's non-obvious the `host_future` never is allowed to fail.